### PR TITLE
Prepare 1.13.0 release

### PR DESCRIPTION
**Pull request by arturbosch**

### CODE REVIEW

**Review by BraisGabin — APPROVED** [review]

**Comment by arturbosch** [comment]
I'm not sure if jcenter just needs again some time after failing to upload so many files or if https://github.com/detekt/detekt/pull/3006 broke uploading some artifacts to the gradle plugin portal.

Need to investigate how to exclude publishing `md5` hashes. One hash should be enough (sha1).

**Review by schalkms — APPROVED** [review]
Agree @arturbosch

**Comment by arturbosch** [comment]
I republished the Gradle plugin.
Jcenter sync still needs some time due to too many failing uploads :/

**Comment by arturbosch** [comment]
Releases is up and in the releases repository. Should be up on maven central in the next minutes/hours.
CI will fail until then.

File: buildSrc/src/main/kotlin/packaging.gradle.kts [hunk 1]
```diff
@@ -8,6 +8,29 @@ plugins {
...
+}
+
+val sonatypeUsername: String? = findProperty("sonatypeUsername")
+    ?.toString()
+    ?: System.getenv("MAVEN_CENTRAL_USER")
+val sonatypePassword: String? = findProperty("sonatypePassword")
+    ?.toString()
+    ?: System.getenv("MAVEN_CENTRAL_PW")
+
+// still needed for artifactory snapshot publishing
```
Comment (cortinico) [inline buildSrc/src/main/kotlin/packaging.gradle.kts:21]: How about we move to use to the Nexus Snapshot repository? So we can remove `id("com.jfrog.artifactory")` and we don't have to handle double credentials 👌

Comment (arturbosch) [reply buildSrc/src/main/kotlin/packaging.gradle.kts:21]: Yep, that would be good. I saved it for another PR ^^.
Feel free to pick it up if you have time. I will have more time starting next week I think.

**Pull request merged**